src/rai/rai/cli/rai_cli.py

```diff
@@ -35,42 +41,80 @@ def parse_whoami_package():
         "documentation_root", type=str, help="Path to the root of the documentation"
     )
     parser.add_argument(
-        "output",
+        "--output",
         type=str,
+        required=False,
+        default=None,
         help="Path to the output directory",
     )
     args = parser.parse_args()
+    save_dir = args.output if args.output is not None else args.documentation_root
+
+    def build_docs_vector_store():
+        logger.info("Building the robot docs vector store...")
+        faiss_index = FAISS.from_documents(docs, OpenAIEmbeddings())
```

Comment:
Same here, we should avoid hard OpenAI dependency and make it interchangeable with OpenSource solution

Reply:
Done